flink-python/setup.py

```diff
@@ -62,6 +62,10 @@
 in_flink_source = os.path.isfile("../flink-java/src/main/java/org/apache/flink/api/java/"
                                  "ExecutionEnvironment.java")
 
+# Due to changes in FLINK-14008, the licenses directory and NOTICE file may not exist in
+# build-target folder. Just ignore them in this case.
+copy_licenses = None
```

Comment:
@WeiZhong94 Thanks a lot for the quick fix! Looks good to me except the variable name, i.e., change `copy_licenses` to `contains_licenses`?